docs/notebooks/pydax-mvp-demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>I expanded the example in the first markdown cell. There's no clean way to run it as a code cell since we need to install pydax before running it so I just embedded the output along with the code as markdown. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/pydax/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>